src/content-steering-controller.js

```diff
@@ -413,13 +408,31 @@ export default class ContentSteeringController extends videojs.EventTarget {
   }
 
   /**
-   * clears all pathways from the available pathways set
+   * Clears all pathways from the available pathways set
    */
   clearAvailablePathways() {
     this.availablePathways_.clear();
   }
 
+  /**
+   * Removes a pathway from the available pathways set.
+   */
   excludePathway(pathway) {
     return this.availablePathways_.delete(pathway);
   }
+
+  /**
+   * Checks the refreshed DASH manifest content steering tag for changes.
+   *
+   * @param {string} baseURL new steering tag on DASH manifest refresh
+   * @param {Object} newTag the new tag to check for changes
+   * @return a true or false whether the new tag has different values
+   */
+  didDASHTagChange(baseURL, newTag) {
+    return !newTag && this.steeringManifest.reloadUri ||
```

Comment:
Out of curiosity, why are we returning true if there is no new tag?

Reply:
Only if there's no new tag _and_ the `reloadUri` is set. So the case where we go from having any content steering tag (since `reloadUri` is required), to none.